manifests/certs.pp

```diff
@@ -1,3 +1,9 @@
+# @summary This class creates a set of certificates in /etc/${service}. These certificates are used when configuring grafana to use
+#   ssl and to connect to PE postgres.  The certificates are based on the agent's own Puppet certificates. 
```

Comment:
`@summary` is supposed to be limited to 140 characters. I suggest replacing this header with the following:

```markdown
# @summary This class creates a certificates for Grafana and for connecting to PE Postgres.
#
# This class creates a set of certificates in /etc/${service}. These certificates
# are used when configuring Grafana to use SSL and to connect to PE Postgres.
# The certificates are based on the agent's own Puppet certificates.
```